Client/Frontend/Popover/PopoverController.swift

```diff
@@ -345,36 +345,39 @@ extension PopoverController {
 extension PopoverController: BasicAnimationControllerDelegate {
     
     func animatePresentation(context: UIViewControllerContextTransitioning) {
-        guard let viewController = context.viewController(forKey: .from), let popoverContext = presentationContext else {
+        guard let popoverContext = presentationContext else {
             context.completeTransition(false)
             return
         }
         
         context.containerView.addSubview(view)
         
-        switch containerView.arrowDirection {
-        case .down:
-            containerView.bottomAnchor.constraint(equalTo: popoverContext.originView.topAnchor, constant: -arrowDistance).isActive = true
-        case .up:
-            containerView.topAnchor.constraint(equalTo: popoverContext.originView.bottomAnchor, constant: arrowDistance).isActive = true
+        let originViewFrame = context.containerView.convert(popoverContext.originView.frame, from: popoverContext.originView.superview)
```

Comment:
Once the view is added as subview at line 353, we can use the view itself for constraint purpose. Current implementation puts constraints between views which are not direct superview/subview. This causes the view lifecycle update calls on the topmost view as the constraints attached to the current view gets updated. Simple replacement of context.containerView with just view, will do it.